ztp/source-crs/SriovVrbClusterConfig.yaml

```diff
@@ -0,0 +1,20 @@
+apiVersion: sriovvrb.intel.com/v1
+kind: SriovVrbClusterConfig
+metadata:
+  name: config
+  namespace: vran-acceleration-operators 
+  annotations:
+    ran.openshift.io/ztp-deploy-wave: "10"
+spec:
+  drainSkip: $drainSkip # true if SNO, false by default
+  priority: 1
+  nodeSelector:
+    node-role.kubernetes.io/master: ""
```

Comment:
Usually we'd have this being `node-role.kubernetes.io/$mcp: ""` so it can be filled in for master or worker as appropriate; Should this be `worker` for MNO setups?  Or is it always `master`?

Reply:
good catch.  The customer could very well have MNO setups representative of Centralized RAN (C-RAN) where the DU is a pooled configuration.  I'll make some changes here to reflect this and put a comment to use `master` for SNO.

Reply:
fixed.